src/view/com/util/text/Text.tsx

```diff
@@ -74,49 +75,45 @@ export function Text({
     // @ts-ignore
     if (flattened.fontSize) {
       // @ts-ignore
-      flattened.fontSize = flattened.fontSize * fonts.scaleMultiplier
+      flattened.fontSize = Math.round(
+        // @ts-ignore
+        flattened.fontSize * fonts.scaleMultiplier,
+      )
     }
 
-    const shared = {
-      uiTextView: true,
+    return {
+      uiTextView: selectable && isIOS,
       selectable,
       style: flattened,
+      dataSet: Object.assign({tooltip: title}, dataSet || {}),
```

Comment:
this is now always assigned but used to be skipped under iOS && selectable, right? seems fine either way cause it’s probably web only

Reply:
Yea, it should definitely be web only. Actually may make sense to _only_ add the prop for `isWeb` anyway.